webapp/channels/src/components/rhs_card_header/rhs_card_header.tsx

```diff
@@ -61,47 +61,47 @@ class RhsCardHeader extends React.PureComponent<Props> {
         case RHSStates.SEARCH:
         case RHSStates.MENTION:
             backToResultsTooltip = (
-                <Tooltip id='backToResultsTooltip'>
+                <WithTooltip id='backToResultsTooltip'>
                     <FormattedMessage
                         id='rhs_header.backToResultsTooltip'
                         defaultMessage='Back to search results'
                     />
-                </Tooltip>
+                </WithTooltip>
             );
             break;
         case RHSStates.FLAG:
             backToResultsTooltip = (
-                <Tooltip id='backToResultsTooltip'>
+                <WithTooltip id='backToResultsTooltip'>
                     <FormattedMessage
                         id='rhs_header.backToFlaggedTooltip'
                         defaultMessage='Back to saved messages'
                     />
-                </Tooltip>
+                </WithTooltip>
             );
             break;
         case RHSStates.PIN:
             backToResultsTooltip = (
-                <Tooltip id='backToResultsTooltip'>
+                <WithTooltip id='backToResultsTooltip'>
                     <FormattedMessage
                         id='rhs_header.backToPinnedTooltip'
                         defaultMessage='Back to pinned messages'
                     />
-                </Tooltip>
+                </WithTooltip>
             );
             break;
         }
 
         const closeSidebarTooltip = (
-            <Tooltip id='closeSidebarTooltip'>
+            <WithTooltip id='closeSidebarTooltip'>
```

Comment:
We dont have to replace Tooltip component with WithTooltip but only for OverlayTrigger we need to do that for this one
```suggestion
            
```
```
const closeSidebarTooltip = <FormattedMessage
                    id='rhs_header.closeSidebarTooltip'
                    defaultMessage='Close'
                />
```

Reply:
ok Let me try to push that by tomorrow